src/component/label.js

```diff
@@ -15,13 +15,15 @@ const balanceStyles = StyleSheet.create({
   numeral: {
     fontFamily: 'WorkSans ExtraLight',
     fontSize: font.sizeXXXL,
-    lineHeight: font.lineHeightXXXL,
+    lineHeight: null,
+    marginLeft: 10,
+    marginRight: 10,
```

Comment:
The home screen already has 20 padding on the left and right. Margin shouldn't be required.

Reply:
fixed, thx